lib/cron/fade-rate.ts

```diff
@@ -22,7 +29,8 @@ async function main(metrics: MetricsLogger) {
     name: 'FadeRate',
     serializers: Logger.stdSerializers,
   });
-  const webhookProvider = new S3WebhookConfigurationProvider(log, `${WEBHOOK_CONFIG_BUCKET}-prod-1`, PRODUCTION_S3_KEY);
+  const s3Key = process.env['stage'] === STAGE.BETA ? BETA_S3_KEY : PRODUCTION_S3_KEY;
+  const webhookProvider = new S3WebhookConfigurationProvider(log, `${WEBHOOK_CONFIG_BUCKET}-prod-1`, s3Key);
```

Comment:
nit: `${WEBHOOK_CONFIG_BUCKET}-${stage}-1`? but at the same time we disabled fade rate cron for beta, so really it doesn't matter